splash/render_options.py

```diff
@@ -20,14 +20,15 @@ def __init__(self, data):
         self.data = data
 
     @classmethod
-    def fromrequest(cls, request):
+    def fromrequest(cls, request, max_timeout=defaults.MAX_TIMEOUT):
```

Comment:
It seems that setting this option to default.MAX_TIMEOUT here doesn't make sense, because the default value is already set in server.py. The advantage of not setting it here is that if you forget to pass this variable you'll get an exception instead of a wrong value.